web/src/ui/commons.js

```diff
@@ -0,0 +1,29 @@
+const Select = (props) => {
+    return (
+		<form className="max-w-sm mx-auto">
+            <select id="{props.label}" className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500">
+                <option defaultValue>Select {props.label}</option>
+                {
+                    props.options.map((option, index) => (
+                        <option key={index} value={option.toLowerCase()}>
```

Comment:
toLowerCase() ist in der allgemeinen Komponente keine gute Idee, das sollte immer scho richtig reingegeben werden